blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React, { Component, PropTypes } from 'react';
+import { Icon } from 'react-material-icons-blue';
+import { ReadableDate } from '@jenkins-cd/design-language';
+import { LiveStatusIndicator } from '@jenkins-cd/design-language';
+import { TimeDuration } from '@jenkins-cd/design-language';
+import moment from 'moment';
+
+const { object, func } = PropTypes;
+
+class RunDetailsHeader extends Component {
+    handleAuthorsClick() {
+        if (this.props.onAuthorsClick) {
+            this.props.onAuthorsClick();
+        }
+    }
+
+    handleOrganizationClick() {
+        if (this.props.onOrganizationClick) {
+            this.props.onOrganizationClick();
+        }
+    }
+
+    handleNameClick() {
+        if (this.props.onNameClick) {
+            this.props.onNameClick();
+        }
+    }
+
+    render() {
+        const {
+            data: {
+                id,
+                name,
+                organization,
+                pipeline,
+                changeSet,
+                result,
+                state,
+                durationInMillis,
+                endTime,
+                startTime,
+                estimatedDurationInMillis,
+                commitId,
+
+            },
+        } = this.props;
+
+        // Grab author from each change, run through a set for uniqueness
+        // FIXME-FLOW: Remove the ":any" cast after completion of https://github.com/facebook/flow/issues/1059
+        const authors = [...(new Set(changeSet.map(change => change.author.fullName)):any)];
+        const status = result === 'UNKNOWN' ? state : result;
+        const running = status === 'RUNNING';
+        const durationMillis = !running ?
+            durationInMillis :
+            moment().diff(moment(startTime));
```

Comment:
I assume this is just the same as `Date.now() - startTime`, or is there some advantage of using `moment`? If there's not, then it seems like a bit of overkill + an unneeded module dep.